src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -112,22 +111,38 @@ public void Ctor_ExpectedDefaultPropertyValues()
                 Assert.NotNull(cookies);
                 Assert.Equal(0, cookies.Count);
                 Assert.Null(handler.Credentials);
-                Assert.Equal(50, handler.MaxAutomaticRedirections);
-                Assert.False(handler.PreAuthenticate);
+                Assert.NotNull(handler.Properties);
                 Assert.Equal(null, handler.Proxy);
                 Assert.True(handler.SupportsAutomaticDecompression);
-                Assert.True(handler.SupportsProxy);
-                Assert.True(handler.SupportsRedirectConfiguration);
                 Assert.True(handler.UseCookies);
-                Assert.False(handler.UseDefaultCredentials);
+
+                if (!PlatformDetection.IsUap) // TODO https://github.com/dotnet/corefx/issues/21510
```

Comment:
I moved the #21510 related issue into its own test.

The other one, related to .NET Framework, is more problematic and that issue number is now closed. I left the Assert where it is and just removed the comment. Most likely, this will stay as-is.